### DISCUSSION TITLE
Word wrap

### DISCUSSION THREAD

**Discussion by MasterH0PP3R** [kefir500/apk-editor-studio#108] [Ideas]
Hi All

Can you do word wrap as most of the text going off the screen, Thanks

**Comment by kefir500** [comment]
Hi @MasterH0PP3R,
Do you mean word wrapping in the code editor?

**Reply by MasterH0PP3R** [reply]
Hi yes in the code editor, it runs off the page and hard to follow when going back and forth

**Reply by kefir500** [reply]
Sounds like a good [feature request](https://github.com/kefir500/apk-editor-studio/issues/109) for the upcoming APK Editor Studio v1.8.0.

**Reply by MasterH0PP3R** [reply]
Ya would be nice to see it added

**Comment by kefir500** [comment]
Implemented in 3bff495.

This feature will be present in the next APK Editor Studio release.
However, you can already check out the developer build [here](https://github.com/kefir500/apk-editor-studio/releases/tag/development).
It is available under the `Tab` → `Word Wrap`.

Please, let me know if you experience any issues with code folding while the word wrapping is enabled.

**Reply by MasterH0PP3R** [reply]
Thanks, I'll check it out, and let you know

**Comment by Ibuprophen** [comment]
Just a quick note that I (personally) use a free software called "Notepad++" for Editing. It's great and very easy to use. 

Just a suggestion... :nerd_face: 


~Ibuprophen

**Reply by MasterH0PP3R** [reply]
Hi ya I use that also which is fine, but be nice to be able to do all from within the program thanks.

**Reply by Ibuprophen** [reply]
Oops... I just remembered that this is a Notepad++ specific feature request, but I use a Plug-in called `TextFX` that has a handful of other features, but I actually only use the 'Word Wrap' feature in it. I think the `TextFX` may still be available only for the 32-bit version of the Notepad++ Software, but I hadn't re-checked this in some time as I don't really require the 64-bit version (yet).

There's the following location to locate this plug-in... 

https://sourceforge.net/projects/npp-plugins/files/TextFX/

... or even the following ftp style URL... 

http://textfx.no-ip.com/textfx/

... but you can always use the following repository to locate that one (as well as other ones you may be interested in). 

https://github.com/notepad-plus-plus/nppPluginList

As you may already know, just simply grab the Plug-in and Install it via the Notepad++ `Plugin Menu`. 

There's a few places where this Plug-in is discussed, but here's one. 

https://superuser.com/questions/365225/how-do-i-wrap-lines-in-notepad-after-n-symbols

I do hope that this helps you out with this situation and wish you the best of luck! 

~Ibuprophen
